test/es-module/test-esm-module-not-found-commonjs-hint.mjs

```diff
@@ -0,0 +1,34 @@
+import '../common/index.mjs';
+import { fixturesDir } from '../common/fixtures.mjs';
+import { ok } from 'assert';
+import { spawn } from 'child_process';
+import { execPath } from 'process';
+
+[
+  {
+    input: 'import "./print-error-message"',
+    // Did you mean to import ../print-error-message.js?
+    expected: ' ../print-error-message.js?'
```

Comment:
Is this a bug? Shouldn’t the error message say `Did you mean to import ./print-error-message.js?` (as in, `./` rather than `../`)?

The same was present in the previous test, so I think that fixing the bug (if this is one) is outside the scope of this PR, but I just wanted to ask if this _is_ a bug that we should file an issue for.